client/lib/wpcom-undocumented/lib/undocumented.js

```diff
@@ -2043,6 +2043,21 @@ Undocumented.prototype.startExport = function( siteId, advancedSettings, fn ) {
 };
 
 /**
+ * Start an export from a Jetpack site
+ *
+ * @param {int}       siteId            The site ID
+ * @param {Object}    advancedSettings  Advanced export configuration
+ * @param {Function}  fn                The callback function
+ * @returns {Promise}                   A promise that resolves when the export started
+ */
+Undocumented.prototype.startJetpackExport = function( siteId, advancedSettings, fn ) {
+	return this.wpcom.withLocale().req.post( {
+		apiVersion: '1.1',
+		path: `/sites/${ siteId }/export`
```

Comment:
We also have a `GET /sites/{ siteId }/export/{ exportId }` endpoint which returns information about the current export (if any) in the async .com exporter. Perhaps if this is a different mechanism, we should namespace to avoid confusion? How about:

`POST /sites/${ siteId }/export/jetpack`

Alternatively, what do you think about using the existing `POST /sites/${ siteId }/export/start` endpoint but make Calypso able to handle both a sync/async response? Eg, the endpoint could return a 'complete' status immediately (sync), or it could return a 'in-progress' response (async) as it does now.

That way Calypso wouldn't really need to know (I think) if its dealing with a Jetpack site or a .com site; it can hit the same endpoint in either case.